.github/workflows/update_disabled_tests.yml

```diff
@@ -19,15 +19,28 @@ jobs:
       - name: Print file
         run: |
           cat disabled-tests.json
+          cat disabled-tests-condensed.json
       - name: Push file to test-infra repository
         uses: dmnemec/copy_file_to_another_repo_action@eebb594efdf52bc12e1b461988d7254322dac131
         env:
-           API_TOKEN_GITHUB: ${{ secrets.GITHUB_TOKEN }}
+          API_TOKEN_GITHUB: ${{ secrets.GITHUB_TOKEN }}
         with:
-          source_file: 'disabled-tests.json'
-          destination_repo: 'pytorch/test-infra'
-          destination_folder: 'stats'
+          source_file: "disabled-tests.json"
```

Comment:
It seems silly that the action doesn't allow multiple files. There seems to be a work around (unconfirmed) https://github.com/dmnemec/copy_file_to_another_repo_action/issues/31.  But there are only 2 files here, so I'm happy to just copy paste :)

Reply:
my hope is to eventually get rid of the non condensed version, but i dont want to do that just yet because pytorch still uses it right now

Reply:
Ah I see. That makes sense